docs/semgrep-supply-chain/license-scanning.md

```diff
@@ -0,0 +1,115 @@
+---
+slug: license-scanning
+append_help_link: true
+title: License scanning 
+hide_title: true
+description: "Semgrep Supply Chain can detect and list a package's license. Prevent or exempt certain packages from being used based on its license."
+tags:
+  - Team & Enterprise tier
+  - Semgrep Supply Chain
+---
+
+# License scanning
```

Comment:
We made a decision to call it 'License Compliance' instead

Reply:
Fixed. Thank you.